backend/services/prisonerProfileService.js

```diff
@@ -104,6 +127,10 @@ module.exports = ({ elite2Api, keyworkerApi, oauthApi, dataComplianceApi, pathfi
         pathfinderUrl && pathfinderDetails && `${pathfinderUrl}nominal/${String(pathfinderDetails.id)}`,
       showPathfinderReferButton: Boolean(!pathfinderDetails && isPathfinderUser),
       pathfinderReferUrl: pathfinderUrl && `${pathfinderUrl}refer/offender/${offenderNo}`,
+      canViewSocLink: socEnabled && canViewSocLink,
+      socProfileUrl: socEnabled && socUrl && socDetails && `${socUrl}nominal/${String(socDetails.id)}`,
```

Comment:
we were trying on all our new endpoints to have the `/` in the URL instead.  That way the URLs read better so would read as `${socUrl}/nominal` instead